manifests/conf.pp

```diff
@@ -19,6 +19,8 @@
   $proxyuser       = undef,
   $sources         = undef,
   $syncsources     = undef,
+  $owner           = undef,
```

Comment:
why are these set to undef? Shouldn't they stay root:root?

Reply:
This is to follow how the other parameters are defined. But ideally it should be `$::sumo::owner` and `$::sumo::group`, to get the default from the `sumo` class, but this should also be done for the rest of the parameters imo.

In any case, if not provided, puppet will default to root normally.